examples/ownSteps/overrideSteps.js

```diff
@@ -14,7 +14,7 @@ MySteps.prototype.openApp = async function () {
     console.log(this._appUrl);
 };
 
-global.SS = new MySteps;
+global.SS = new MySteps();
```

Comment:
well, in js, as I know, variant without parenthesis when no arguments, is working too. But ok, let it be more explicit.